packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -476,6 +488,36 @@ describe('RPC Server Acceptance Tests', function() {
         });
     });
 
+    describe('eth_getCode', () => {
```

Comment:
Since we will be using the `basic` contract only for the `eth_getCode` rpc calls I think that it is better to add a `before` here and deploy the contract within this scope